crates/store/re_video/src/decode/h264_sps.rs

```diff
@@ -0,0 +1,329 @@
+use super::YuvPixelLayout;
+
+/// Sequence Parameter Set for h264 video
+///
+/// Semantics are defined in [ITU-T H.264 (04/2017)](http://wikil.lwwhome.cn:28080/wp-content/uploads/2018/08/T-REC-H.264-201704%E8%8B%B1%E6%96%87.pdf)
+#[derive(Debug)]
+#[allow(dead_code)]
+pub struct H264Sps {
```

Comment:
I assume this appears in all H.264 stream, regardless of container format? i.e. that is why it belongs in `re_video`, and not `re_mp4`?

Reply:
Correct, the SPS is part of the h264 spec (as linked in the comment). So this will still be useful when streaming in data as we'll want to pass through the AVCC box (which typically contains the SPS among other things). At least I think at this point that we want to insist on that rather than consuming AnnexB (I recently read up some more on how this works in WebCodecs which btw. is a great blueprint for our mux-free/streaming api  ;-) [.........])